Formula/terraforming.rb

```diff
@@ -0,0 +1,113 @@
+class Terraforming < Formula
+  desc "Export existing AWS resources to Terraform style (tf, tfstate)"
+  homepage "http://terraforming.dtan4.net/"
+  url "https://github.com/dtan4/terraforming.git",
+      :tag => "v0.15.0",
```

Comment:
can we use the source archive tarball?

Reply:
Unfortunately no, as https://github.com/dtan4/terraforming/blob/master/terraforming.gemspec#L17 implies that the gem building process would take place in a git repo.

Reply:
ok